Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -622,7 +622,7 @@ private async void OnDeactivated(object sender, EventArgs e)
                 if (_settings.UseAnimation)
                     await Task.Delay(100);
 
-                if (_settings.HideWhenDeactivated)
+                if (_settings.HideWhenDeactivated && !_viewModel.ExternalPreviewVisible)
```

Comment:
why?

Reply:
This is for when you click away from the mouse it doesnt hide both the search window and preview. Not sure which is user-friendlier, hide when deactivated or do not.